pytest.ini

```diff
@@ -1,5 +1,6 @@
 [pytest]
```

Comment:
for the sake of density i'd put these configs in the pyproject.toml

Reply:
I'm not sure about this. One thing that bothers me with this is that as of now, the standard library has no `toml` parser and so some projects like `flake8` will not support it. Thus, I will have some files "scattered" around in the project. I'll keep the current state for now.